packages/neo-one-client/src/__tests__/provider/neoone/JSONRPCClient.test.js

```diff
@@ -0,0 +1,161 @@
+/* @flow */
+import BigNumber from 'bignumber.js';
+
+import { keys, transactions } from '@neo-one/client-core/src/__data__';
+
+import JSONRPCClient from '../../../provider/neoone/JSONRPCClient';
+import { RelayTransactionError } from '../../../errors';
+
+jest.mock('isomorphic-fetch');
+
+describe('JSONRPCClient', () => {
+  const expected = '10';
+  const jsonProvider = { request: jest.fn(() => Promise.resolve(expected)) };
+  const provider = new JSONRPCClient(jsonProvider);
+  const testCases = [
+    {
+      method: 'getAccount',
+      args: [keys[0].address],
+    },
+    {
+      method: 'getAsset',
+      args: [transactions.register.hash],
+    },
+    {
+      method: 'getBlock',
+      args: [transactions.register.hash],
+    },
+    {
+      method: 'getBlock',
+      args: [0],
```

Comment:
Same here as `getTransactionReceipt`, add a case where `options` is non-null so `watchTimeout` is defined in the snapshot.